include/functions.h

```diff
@@ -2450,18 +2421,19 @@ void Message_FindCreditsMessage(PlayState* play, u16 textId);
 void func_8015E7EC(PlayState* play, UNK_PTR puParm2);
 // void func_8015F8A8(UNK_TYPE4 ctxt);
 
-void ShrinkWindow_SetLetterboxTarget(s32 target);
-s32 ShrinkWindow_GetLetterboxTarget(void);
-void ShrinkWindow_SetLetterboxMagnitude(s32 magnitude);
-s32 ShrinkWindow_GetLetterboxMagnitude(void);
-void ShrinkWindow_SetPillarboxTarget(s32 target);
-s32 ShrinkWindow_GetPillarboxTarget(void);
-void ShrinkWindow_SetPillarboxMagnitude(s32 magnitude);
-s32 ShrinkWindow_GetPillarboxMagnitude(void);
+void Letterbox_SetSizeTarget(s32 target);
+s32 Letterbox_GetSizeTarget(void);
+void Letterbox_SetSize(s32 size);
+s32 Letterbox_GetSize(void);
+void Pillarbox_SetSizeTarget(s32 target);
```

Comment:
```suggestion
s32 ShrinkWindow_Letterbox_GetSize(void);
void ShrinkWindow_Pillarbox_SetSizeTarget(s32 target);
```
Personally I'm fine with how it is right now, but thoughts on naming like this so Letterbox and Pillarbox appear as subsystems to the ShrinkWindow system.

Reply:
The issue is that OoT Decomp has chosen to ignore the evidence and opted to remove the ShrinkWindow system, since OoT's implementation of Shrinkwindow only implements Letterboxing and MM's implementation underwent significant refactoring.

Renaming these `Letterbox` and `Pillarbox` is an attempt to salvage consistency between the games.

Reply:
I feel like just prefixing with `ShrinkWindow` doesn't really harm consistency. Someone from OoT using `Letterbox_GetSize` can still find the same function by searching in `MM` for `Letterbox_GetSize`. They would also get the added bonus of knowing that it is now part of a larger system called `ShrinkWindow`.

Reply:
I simply like the shortness and simplicity of `Letterbox_GetSize` and `Pillarbox_GetSize` on it's own, but I think I'm fine adding a `ShrinkWindow_` prefix to it. Not my preference, but I could roll with it

Reply:
I prefer we if we could keep the `ShrinkWindow` prefix too, since all of those functions are part of that system